src/server.h

```diff
@@ -1509,7 +1509,8 @@ struct redisServer {
     clientMemUsageBucket client_mem_usage_buckets[CLIENT_MEM_USAGE_BUCKETS];
 
     rax *clients_timeout_table; /* Radix tree for blocked clients timeouts. */
-    long fixed_time_expire;     /* If > 0, expire keys against server.mstime. */
+    long call_depth;            /* call() re-entering count.
```

Comment:
pretty sure we don't need this one and can somehow use in_nested_call

Reply:
I thought about using it but hesitated... I have changed to use in_nested_call, thanks for the "pretty sure" ack